### PR TITLE
Implement getting HTML color code for languages

### PR DESCRIPTION
**Pull request by lafriks**
Adds utility function `GetColor(language string) string`

### CODE REVIEW

**Comment by lafriks** [comment]
Is there anything else I need to do to get this merged?

**Comment by creachadair** [comment]
> Is there anything else I need to do to get this merged?

Sorry for the delay. @bzz who maintains this repo is out of the office this week, but should be able to take a look when he gets back. The change looks fine from a functional perspective, but I'd like Alex's feedback on whether it makes sense to include it.

File: internal/code-generator/assets/colors.go.tmpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package data
+
+var LanguagesColor = map[string]string{
```
Comment (bzz) [inline internal/code-generator/assets/colors.go.tmpl]: @lafriks here and below, could you also please make sure all exported functions/variables are documented? Thanks!

**Comment by bzz** [comment]
Sorry for delay, great job @lafriks, LGTM.

It would be also interesting to know if you could share about your use-case for this feature!

**Comment by lafriks** [comment]
@bzz no problems, I'm one of core Gitea contributors and currently implementing repository language statistics bar like in Github, see back-reference above for Gitea PR that is currently wip

**Comment by lafriks** [comment]
Btw thanks for for this great library that made this possible to add that to Gitea ;)

**Pull request merged**